person.rb

```diff
@@ -20,6 +22,10 @@ def validate_name
     @name = @corrector.correct_name(@name)
   end
 
+  def add_rentals(book, date)
+    Rentals.new(date, self, book)
```

Comment:
- [ ] You have a typo calling the class "Rentals " instead of "Rental". Please fix it in order to make the rental object work properly.
- [ ] Also in the class "Book"

Reply:
Oh, okay. Thanks 👍